examples/benchmarks/main.cpp

```diff
@@ -33,28 +36,28 @@
 using namespace tiny_dnn;
 using namespace std;
 
-int main(int argc, char** argv) {
-  CNN_UNREFERENCED_PARAMETER(argc);
-  CNN_UNREFERENCED_PARAMETER(argv);
-  models::alexnet nn;
+int main(int argc, char **argv) {
```

Comment:
I believe aligning `*` and `&` might be misleading. Anyway that was implicit change and not explicit, I think this is included in function parameter aligning